modular_citadel/code/modules/client/loadout/__donator.dm

```diff
@@ -353,3 +356,128 @@
+	name = "Over and Done for"
+	path = /obj/item/storage/box/custom_kit/notsoveterananymore
+	ckeywhitelist = list ("zerickyradurandal","especiallystrange")
+
+/obj/item/storage/box/custom_kit/notsoveterananymore/PopulateContents()
+	new /obj/item/clothing/suit/armor/f13/rangercombat/finlayranger_alt(src)
+	new /obj/item/clothing/head/helmet/f13/ncr/rangercombat/finlayranger_alt(src)
+	new /obj/item/clothing/head/beret/donator_worn_beret(src)
+	new /obj/item/gun/ballistic/rifle/repeater/trail(src)
+	new /obj/item/ammo_box/m44box(src)
+	new /obj/item/ammo_box/a357box(src)
+	new /obj/item/storage/belt/holster/ranger357(src)
+	new /obj/item/melee/classic_baton/telescopic(src)
+	new /obj/item/attachments/scope(src)
+
+/datum/gear/donator/kits/reikafled
+	name = "My Memoir"
+	path = /obj/item/storage/box/custom_kit/reikafled
+	ckeywhitelist = list ("zerickyradurandal")
+
+/obj/item/storage/box/custom_kit/reikafled/PopulateContents()
+	new /obj/item/pda/reika(src)
+	new /obj/item/gun/ballistic/automatic/pistol/n99(src)
+	new /obj/item/ammo_box/magazine/m10mm_adv/simple(src)
+	new /obj/item/ammo_box/magazine/m10mm_adv/simple(src)
+	new /obj/item/clothing/suit/armor/f13/leather_jacket/combat/reika(src)
+/obj/item/pda/reika
+	name = "worn pip-boy"
+	desc = "The RobCo Pip-Boy, This one look rather worn from usages."
+
+/datum/gear/donator/kits/shortrespirator
+	name = "Respirator Kit"
+	path = /obj/item/storage/box/custom_kit/shortrespirator
+	ckeywhitelist = list ("farsightednightlight","raxraus")
+
+/obj/item/storage/box/custom_kit/shortrespirator/PopulateContents()
+	new /obj/item/clothing/mask/gas/sechailer/caligra(src)
+
+/datum/gear/donator/kits/antburger
+	name = "Dunce"
+	path = /obj/item/storage/custom_kit/antburger
+	ckeywhitelist = list ("zerantio")
+
+/obj/item/storage/custom_kit/antburger/PopulateContents()
+	new /obj/item/clothing/accessory/necklace/silvercross(src)
+	new /obj/item/clothing/head/beret/antburger(src)
+
+/datum/gear/donnator/kits/fluffysleepwear
+	name = "Sleeping Equipment"
+	path = /obj/item/storage/custom_kit/fluffycampinglads
+	ckeywhitelist = list ("bonirus")
+
+/obj/item/storage/custom_kit/fluffycampinglads
+	new /obj/item/clothing/shoes/sneakers/white(src)
+	new /obj/item/clothing/under/syndicate/bloodred/sleepytime(src)
+	new /obj/item/clothing/under/sweater(src)
+	new /obj/item/clothing/under/sweater/blue(src)
+	new /obj/item/clothing/under/syndicate/rus_army(src)
+	new /obj/item/clothing/under/misc/pj/red(src)
+	new /obj/item/clothing/under/misc/pj/blue(src)
+	new /obj/item/toy/plush/borgplushie/medihound(src)
+	new /obj/item/gun/ballistic/automatic/c20r/toy/unrestricted/riot(src)
+	new /obj/item/ammo_box/magazine/toy/smgm45/riot(src)
+	new /obj/item/ammo_box/magazine/toy/smgm45/riot(src)
+	new /obj/item/ammo_box/magazine/toy/smgm45/riot(src)
+	new /obj/item/bedsheet/medical(src)
+	new /obj/item/bedsheet/patriot(src)
+	new /obj/item/stack/sheet/cloth/thirty(src)
```

Comment:
Seriously? Starting someone off with 30 sheets of cloth in a "fluff loadout" when the Legion doesn't even start with cotton seeds?